static/css/documentation.css

```diff
@@ -1742,7 +1742,7 @@ Beta tag
     font-size: 20px;
 }
 
-.beta-tag+p:before {
+.beta-tag:before {
     content: "⚠ BETA";
     font-weight: bold;
     display: block;
```

Comment:
let's add this to the end of this section.

```
/* Known Issues Heading */
.beta-tag p:nth-child(2) {
  font-size: 16px;
  margin-bottom: 5px;
}

/* Known Issues List */
.beta-tag ul {
  margin-top: 0px;
  font-size: 14px;
}
```

It makes the text a little better to read.
![TestKnownIssues](https://user-images.githubusercontent.com/23404615/114937977-20116200-9e04-11eb-933e-f3713edaefa4.png)

Reply:
[updated](https://github.com/cerner/fhir.cerner.com/pull/608/commits/f92dcae879184339b59668e9436873b6d2738d2a)

Reply:
can u update the screenshot of this on slot patch please?

Reply:
added the screenshot https://github.com/cerner/fhir.cerner.com/pull/608#issue-610636874